Holovibes/mainwindow.ui

```diff
@@ -1162,6 +1198,9 @@
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="PaccLabel">
+             <property name="toolTip">
+              <string>Interval to include after z (max #img)</string>
```

Comment:
Number of frequencies kept by the FFT in the time transformation algorithm : the range of frequencies kept is [z, z + width].